DataScience/Experimentation.py

```diff
@@ -201,36 +201,30 @@ def parse_cb_types(val):
     return cb_types
 
 def generate_predictions_files(log_fp, policies):
-
     predictions_files = []
     data = {}
     data['policies'] = []
-    print('Generating predictions files (using --cb_explore_adf) for {} policies:'.format(len(policies)))
-    for name,policy in policies:
-        policy_command =  policy.full_command.replace('--cb_adf', '--cb_explore_adf --epsilon 0.2')
-        data['policies'].append({
-            'name':name,
-            'arguments': re.sub(r'(-c|-d\s[\S]*|vw)\s', '', policy_command),
-            'loss': policy.loss
+    for name, policy in policies:
+        pred_fp = log_fp + '.' + name + '.pred'
+        predictions_files.append(pred_fp)
+        cmd = policy.full_command.replace('--cb_adf', '--cb_explore_adf --epsilon 0.2') + ' -p ' + pred_fp + ' -P 100000 '
+        p = Popen(cmd.split(' '), stdout=PIPE, stderr=PIPE)
```

Comment:
Instead of using PIPE, I suggest we use 
`results = check_output(command.full_command, stderr=STDOUT).decode("utf-8")`
as done for cb_adf, so that we keep things consistent in this file.
If PIPE is a better more secure way, then we should update the command in both places.